### DISCUSSION TITLE
所见即所得和卡片并列显示功能？

### DISCUSSION THREAD

**Discussion by nightlye** [tiddly-gittly/TidGi-Desktop#413] [Q&A]
太记确实很好用，但编辑方面还是很喜欢siyuan那种所见即所得的编辑方式，另外siyuan和vscode那种可以把多个标签页并列显示的功能也很好用，不知道太记能不能把卡片也类似地并列显示呢？

**Comment by linonetwo** [comment]
所见即所得有插件呀，cpl 搜 slate-write

并列显示也有插件呀，论坛搜搜看

**Comment by linonetwo** [comment]
不对，太记不是自带了所见即所得插件吗，不过目前没有教程，所以希望更多群友去 https://github.com/tiddly-gittly/TidGi-Official-Website 更新太记用法教程
